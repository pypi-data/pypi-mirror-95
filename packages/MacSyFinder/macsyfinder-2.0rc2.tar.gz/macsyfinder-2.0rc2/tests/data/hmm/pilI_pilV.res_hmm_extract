PSAE001c01_046080	PSAE001c01	4608	pilI_pilV	T4P	4.4e-66	221.1	1.000000
