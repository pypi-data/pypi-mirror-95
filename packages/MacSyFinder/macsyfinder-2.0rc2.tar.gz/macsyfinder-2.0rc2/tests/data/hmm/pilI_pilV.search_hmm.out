# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.0 (March 2010); http://hmmer.org/
# Copyright (C) 2010 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  ./profiles/pilI_pilV.fasta-aln_edit.hmm
# target sequence database:        datatest/prru_psae.001.c01.fasta
# output directed to file:         ./datatest/res_search/pilI_pilV.search_hmm.out
# sequence reporting threshold:    E-value <= 1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       pilI_pilV.tab.seqid  [M=171]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description
    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------
    3.9e-66  221.3   0.0    4.4e-66  221.1   0.0    1.0  1  PSAE001c01_046080 D ATG TGA 5098709 5099266 Valid pilV 558 _
    6.5e-05   22.0   1.9    7.2e-05   21.9   0.6    1.5  2  PSAE001c01_031360 C ATG TGA 3479335 3479724 Valid xcpV 390 _
     0.0016   17.5   0.0     0.0019   17.2   0.0    1.2  1  PSAE001c01_027000 C ATG TGA 3020504 3020929 Valid PA2673 426
     0.0046   16.0   3.2      0.007   15.4   2.2    1.3  1  PSAE001c01_006890 C ATG TGA 738111 738485 Valid PA0680 375 _
  ------ inclusion threshold ------
       0.01   14.8   0.7      0.014   14.4   0.5    1.2  1  PSAE001c01_006860 C ATG TGA 736456 737097 Valid PA0677 642 _
      0.023   13.7   0.2      0.029   13.4   0.1    1.2  1  PSAE001c01_031370 C ATG TGA 3479721 3480239 Valid xcpU 519 _
      0.092   11.8   0.1       0.12   11.4   0.1    1.2  1  PSAE001c01_006900 D ATG TGA 738719 739168 Valid PA0681 450 _


Domain annotation for each sequence (and alignments):
>> PSAE001c01_046080  D ATG TGA 5098709 5099266 Valid pilV 558 _PA4551_NP_253241.1_ PA4551 1 5098709 5099266 | type 4 fi
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  221.1   0.0   3.7e-69   4.4e-66       1     171 []      14     173 ..      14     173 .. 1.00

  Alignments for each domain:
  == domain 1    score: 221.1 bits;  conditional E-value: 3.7e-69
  pilI_pilV.tab.seqid   1 GltliEvLvavlilaigvLgllslllksvqstreaeerakasllaqnllerlranpsdsakaaYavktgaattsttlsaggtdfpdasaa 90 
                          G+++iEvLva+l+++igvLg++++++k++q+t++++er+ka++l++nlle++ra+p    ka+Y+vk+++at+s++++a+g++fp+a+++
    PSAE001c01_046080  14 GFSMIEVLVALLLISIGVLGMIAMQGKTIQYTADSVERNKAAMLGSNLLESMRASP----KALYDVKDQMATQSDFFKAKGSAFPTAPSS 99 
                          9*******************************************************....****************************** PP

  pilI_pilV.tab.seqid  91 caaknalveadLkewseelknalpgaagvivsdyavcrsssgfssncdnkandilevkvaWvglskadsgasvaasevsaD 171
                          c++++++++++L++w+e++kn+lpga+++++sdy++crss++  ++cd+k +++le+++aW+g    ++ga+v+a+++saD
    PSAE001c01_046080 100 CTPLPDAIKDRLGCWAEQVKNELPGAGDLLKSDYYICRSSKP--GDCDGK-GSMLEIRLAWRG----KQGACVNAADSSAD 173
                          ******************************************..******.************....*************9 PP

>> PSAE001c01_031360  C ATG TGA 3479335 3479724 Valid xcpV 390 _PA3099_NP_251789.1_ PA3099 1 3479335 3479724 | general s
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   21.9   0.6     6e-08   7.2e-05       1      56 [.       6      61 ..       6      77 .. 0.92
   2 ?   -1.0   0.0      0.65   7.8e+02     132     153 ..      81     102 ..      57     105 .. 0.61

  Alignments for each domain:
  == domain 1    score: 21.9 bits;  conditional E-value: 6e-08
  pilI_pilV.tab.seqid  1 GltliEvLvavlilaigvLgllslllksvqstreaeerakasllaqnllerlranp 56
                         G+tl+EvLva+ i a+   ++ls  ++s q     e ++ a  +a n l+ l++ +
    PSAE001c01_031360  6 GFTLLEVLVALAIFAMVAASVLSASARSLQNASRLEDKTLAMWIADNRLNELQLEQ 61
                         9**********************************************999998765 PP

  == domain 2    score: -1.0 bits;  conditional E-value: 0.65
  pilI_pilV.tab.seqid 132 gfssncdnkandilevkvaWvg 153
                          ++ +++d +a + ++  + Wv 
    PSAE001c01_031360  81 EWRTQVDSTAEQDMRRVIVWVA 102
                          4566788888444444456765 PP

>> PSAE001c01_027000  C ATG TGA 3020504 3020929 Valid PA2673 426 _NP_251363.1_ PA2673 1 3020504 3020929 | type II secret
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   17.2   0.0   1.6e-06    0.0019       1      53 [.       7      62 ..       7      97 .. 0.90

  Alignments for each domain:
  == domain 1    score: 17.2 bits;  conditional E-value: 1.6e-06
  pilI_pilV.tab.seqid  1 GltliEvLvavlilai...gvLgllslllksvqstreaeerakasllaqnllerlr 53
                         G+tl+E+ va+ +la+   ++ + l+   +s+q ++ ae r  ++ +a   ler++
    PSAE001c01_027000  7 GFTLLEAVVALTLLAVvggALFAWLNSAFRSMQRVEAAELRIETARVAMAYLERMN 62
                         9**********999987767999******************************997 PP

>> PSAE001c01_006890  C ATG TGA 738111 738485 Valid PA0680 375 _NP_249371.1_ PA0680 1 738111 738485 | HxcV pseudopilin  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   15.4   2.2   5.9e-06     0.007       1      22 [.       9      30 ..       9      38 .. 0.90

  Alignments for each domain:
  == domain 1    score: 15.4 bits;  conditional E-value: 5.9e-06
  pilI_pilV.tab.seqid  1 GltliEvLvavlilaigvLgll 22
                         G+tliEvLva+ i+ai++ +++
    PSAE001c01_006890  9 GFTLIEVLVALAIVAIALAAAI 30
                         9****************98887 PP

>> PSAE001c01_006860  C ATG TGA 736456 737097 Valid PA0677 642 _NP_249368.1_ PA0677 1 736456 737097 | HxcW pseudopilin  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   14.4   0.5   1.2e-05     0.014       1      50 [.      16      68 ..      16      76 .. 0.77

  Alignments for each domain:
  == domain 1    score: 14.4 bits;  conditional E-value: 1.2e-05
  pilI_pilV.tab.seqid  1 GltliEvLvavlilai.gvLgllsl..llksvqstreaeerakasllaqnlle 50
                         G+tliEv+va++++ai ++++  +l  ++++ +  ++++e+  a+l a n le
    PSAE001c01_006860 16 GFTLIEVMVAIMLMAIvSLMAWRGLdsIARASAHLEDSTEQGAALLRALNQLE 68
                         9*********9998862566665554478999999999999888776666655 PP

>> PSAE001c01_031370  C ATG TGA 3479721 3480239 Valid xcpU 519 _PA3100_NP_251790.1_ PA3100 1 3479721 3480239 | general s
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   13.4   0.1   2.4e-05     0.029       1      45 [.       6      51 ..       6      84 .. 0.78

  Alignments for each domain:
  == domain 1    score: 13.4 bits;  conditional E-value: 2.4e-05
  pilI_pilV.tab.seqid  1 GltliEvLvavlilaigv.Lgllslllksvqstreaeerakaslla 45
                         G+tliE +v ++i+++ + L++ls    s   + ++e +  a l++
    PSAE001c01_031370  6 GFTLIELMVVMVIISVLIgLAVLSTGFASTSRELDSEAERLAGLIG 51
                         9*************99887888888888888888888877777665 PP

>> PSAE001c01_006900  D ATG TGA 738719 739168 Valid PA0681 450 _NP_249372.1_ PA0681 1 738719 739168 | HxcT pseudopilin  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   11.4   0.1   9.8e-05      0.12       1      33 [.      18      51 ..      18      84 .. 0.65

  Alignments for each domain:
  == domain 1    score: 11.4 bits;  conditional E-value: 9.8e-05
  pilI_pilV.tab.seqid  1 GltliEvLvavlilaigvLgll...slllksvqstr 33
                         G+tliE++v v+il  g+L+++    +l +  q  +
    PSAE001c01_006900 18 GFTLIEIMVVVVIL--GILAAMvvpKVLDRPDQARA 51
                         9*****99988876..66665411134444444444 PP



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (171 nodes)
Target sequences:                       8332  (2926593 residues)
Passed MSV filter:                       338  (0.0405665); expected 166.6 (0.02)
Passed bias filter:                      273  (0.0327652); expected 166.6 (0.02)
Passed Vit filter:                        36  (0.00432069); expected 8.3 (0.001)
Passed Fwd filter:                         7  (0.000840134); expected 0.1 (1e-05)
Initial search space (Z):               8332  [actual number of targets]
Domain search space  (domZ):               7  [number of targets reported over threshold]
# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.06
# Mc/sec: 8340.79
//
